File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/adjustech/GraduationDesignTutorAddVo.java
package top.zbeboy.isy.web.vo.graduate.design.adjustech;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2017/5/25.
 */
@Data
public class GraduationDesignTutorAddVo {
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    @NotNull
    private String studentId;
    @NotNull
    @Size(max = 64)
    private String graduationDesignTeacherId;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/adjustech/GraduationDesignTutorAddVo.kt
package top.zbeboy.isy.web.vo.graduate.design.adjustech

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-01-20 .
 **/
open class GraduationDesignTutorAddVo {
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    @NotNull
    var studentId: String? = null
    @NotNull
    @Size(max = 64)
    var graduationDesignTeacherId: String? = null
}